### PR TITLE
Code owners

### PR DESCRIPTION
**Pull request by joecowton1**
## What are you changing?

- adding specific code owners for project directories.

## Why?

- we're currently both unspecific and low on reviewers.

We've added two new teams `@guardian/source` (to review Source repos) and `@guardian/csnx` (to review CSNX config).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,8 +1,20 @@
 # if you would like to be notified about changes in particular part of this repo,
+
 # please add yourself in a PR
 
-* @guardian/client-side-infra
+* @guardian/csnx
+
+libs/@guardian/atoms-rendering/* @guardian/commercial-dev @guardian/client-side-infra
```
Comment (mxdvl) [inline .github/CODEOWNERS:7]: I think @guardian/dotcom-platform  will want to be added to this one

**Review by mxdvl — APPROVED** [review]
This looks great!

File: .github/CODEOWNERS (hunk 1 above)
Comment (joecowton1) [inline .github/CODEOWNERS:7]:
```suggestion
libs/@guardian/atoms-rendering/* @guardian/commercial-dev @guardian/client-side-infra
```

**Comment by joecowton1** [comment]
> This looks great!

Thanks Max, we can always change things going forward but lets merge for now and see how it feels.

**Review by OllysCoding — APPROVED** [review]
lgtm :)

**Pull request merged**